clang/include/clang/Sema/Sema.h

```diff
@@ -4749,6 +4753,15 @@ class Sema final : public SemaBase {
 
   CXXRecordDecl *getStdBadAlloc() const;
   EnumDecl *getStdAlignValT() const;
+  ClassTemplateDecl *getStdTypeIdentity() const;
+  std::optional<QualType> InstantiateSpecializedTypeIdentity(QualType Subject);
```

Comment:
all of these should start with lower case.

Reply:
What is the correct coding style? Over time I've been told upper case, whereas other times lower - in this particular case I followed lower case for the `get` for consistency with the neighboring `get` methods, and upper case for the new one because it didn't have a "consistency with similar methods" rationale and the feedback on other patches seemed to imply that functions are _meant_ to have upper case first letter (I think I actually intentionally capitalized this function before pushing the PR \o/)

I'd prefer lower case start, but I just want to know which it is actually meant to be :D

Reply:
IIRC, our coding standard says camel case:
```
Function names should be verb phrases (as they represent actions), and command-like function should be imperative. The name should be camel case, and start with a lower case letter (e.g. openFile() or isFoo()).```